examples/v2/cloudbuild/python/cloudbuild.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2017 Google Inc. All rights reserved.
```

Comment:
This one failed the same way :( I'm deleting my deployments between so hopefully not a resource conflict?

Reply:
It's probably the same issue as your other deployment.